Documentation/Documentation/style_guide.md

```diff
@@ -0,0 +1,103 @@
+
+## Comprehensive
+**Cover concepts in-full, or not at all.** Describe all of the functionality of a product. Do not omit functionality that you regard as irrelevant for the user. Do not write about what is not there yet. Stay in the current.
+
+## Conformant
+**Describe what you see.** Use explicit examples to demostrate how a feature works. Provide instructions rather than descriptions. Present your information in the order that users experience the subject matter.
+
+**Avoid future tense** (or using the term "will") whenever possible. For example, future tense ("The screen will display...") does not read as well as the present tense ("The screen displays..."). Remember, the users you are writing for most often refer to the documentation while they are using the system, not after or in advance of using the system.  
+Use simple present tense as much as possible. It avoids problems with consequences and time related communications, and is the easiest tense for translation.
+
+**Include (some) examples and tutorials in content.** Many readers look first towards examples for quick answers, so including them will help save these people time. Try to write examples for the most common use cases, but not for everything.
+
+
+## Tone
+**Write in a neutral tone**. Avoid humor, personal opinions, colloquial language and talking down to your reader. Stay factual, stay technical.
+
+**Example**:
+    _The applet is a handy little screen grabber._  
+**Rewrite**:
+    _You use the applet to take screenshots._
+
+**Use active voice (subject-verb-object sequence)** as it makes for more lively, interesting reading. It is more compelling than passive voice and helps to reduce word count. [Examples.](https://examples.yourdictionary.com/examples-of-active-and-passive-voice.html)
+
+**Example**:
+    _The CLI tool creates the boilerplate._  
+**Rewrite**:
+    _The boilerplate is created by the CLI tool._
+
+**Use second person ("you") when speaking to or about the reader**. Authors can refer to themselves in the first person ("I" in single-author articles or "we" in multiple-author articles) but should keep the focus on the reader.
+
+**Avoid sexist language**. There is no need to identify gender in your instructions.
+
+## Formatting
+Use **bold** to emphasize text that is particularly important, bearing in mind that overusing bold reduces its impact and readability.
+
+Use `inline code` for anything that the reader must type or enter. For methods, classes, variables, code elements, files and folders.
+
+Use _italic_ when introducing a word that you will also define or are using in a special way. (Use rarely, and do not use for slang.)
+
+[Hyperlinks](https://en.wikipedia.org/wiki/Hyperlink) should surround the words which describe the link itself. Never use links like "click here" or "this page".
+
+
+Use tips for **practical**, non-essential information.
+{{% notice tip %}}
```

Comment:
It would be good to have examples of how the format of you do this formatting, as this is Hugo specific. Example from Microsoft on what they've done: https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown#note-blocks

Reply:
I was thinking about it but I wanted the style guide to be about  style specifically, I'm working on what I call a "writing guide" that goes more into the nitty gritty of how the text actually looks like, how to use links etc. My idea is that the style guide is something that people can have on their side as a quick reference while writing.